week2/미션/index.css

```
@@ -0,0 +1,174 @@
+@import url('https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css');
+
+/* Global Styles */
+button{
+    background: none;
+    border: none;
+    padding: 0;
+    cursor: pointer;
+    outline: inherit;
+}
+
+/* 헤더 */
+.post-header {
+    width: 100%;
+    position: fixed;
+    display: flex;
+    justify-content: space-between;
+    padding: 14px 16px;
+    box-sizing: border-box;
+    border-bottom: 1px solid #D9D9D9;
+    background-color: #ffff;
+    top: 0px;
+}
+
+.post-header .post-header__icon {
+    display:flex;
+    align-items: center;
+    gap: 16px;
+}
+
+.post-header__icon .local {
+    font-family: Inter;
+    font-size: 18px;
+    font-weight: 700;
+    line-height: 28px;
+    letter-spacing: 0em;
+    text-align: left;
+    color: #000000;
+    margin-right: 6px;
+}
+
+/* 게시글 */
+.post {
+    padding: 0 16px;
+}
+
+.post > section {
+    padding: 16px 0;
+    border-bottom: 1px solid #EEEEEE;
+}
+
+.post > section:first-child {
+    margin-top: 54px;
+}
+
+.post > section:last-child {
+    border-bottom: none;
+    margin-bottom: 54px;
+}
+
+.post__list .list__product {
+    display: flex;
+}
+
+.list__product .list__prd-disc {
+    display: flex;
+    width: calc(100% - 110px);
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.list__product .list__prd-img {
+    width: 110px;
+    height: 110px;
+    border-radius: 4px;
+    margin-right: 16px;
+}
+
+.list__product .prd__name {
+    font-family: pretendard;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 20px;
+    letter-spacing: 0em;
+    text-align: left;
+}
+
+.list__product .prd__local {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 16px;
+    letter-spacing: 0em;
+    text-align: left;
+    color: #8C8C8C;
+    font-family: pretendard;
+}
+
+.list__product .prd__price {
+    font-size: 15px;
+    font-weight: 700;
+    line-height: 22px;
+    letter-spacing: 0em;
+    text-align: left;
+    font-family: pretendard;
+    color: #FF7E36;
+}
+
+.list__product .reaction {
+    display: flex;
+    justify-content: end;
+    align-items: end;
+    gap: 4px;
+}
+
+.reaction .chat{
+    display: flex;
+    gap: 2px;
+}
+
+.reaction .heart{
+    display: flex;
+    gap: 2px;
+}
+
+.reaction .rct {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 16px;
+    letter-spacing: 0em;
+    text-align: left;
+    font-family: pretendard;
+    color: #8C8C8C;
+}
+
+/* 하단 바 */
+.bottom {
+    display: relative;
+    justify-content: space-between;
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    background-color: #fff;
+    padding: 8px 0px 8px 0px;
+    border-top: 1px solid #d9d9d9;
+}
+
+.bottom .bottom__bar {
+    display: flex;
+    justify-content: space-around;
+    width: 100%;
+}
+
+.bottom__bar .bar__info{
+    display: flex;
+    flex-direction: column;
+    gap: 4px;
```

바텀에 메뉴바들이 높이가 약간 안맞는데 
```suggestion
   justify-content: space-between;
```
이렇게 주면 얼추 비슷하게 됩니다! 
button이랑 안에 있는 img들도 높이가 살짝씩 달라서 맞춰주시면 고르게 보일거에요